tests/fs-tree/fs-test.js

```diff
@@ -1812,6 +1783,137 @@ describe('FSTree fs abstraction', function() {
       expect(changes).to.have.property('length', 0);
     });
 
+    describe('srcTree is true', function() {
+      beforeEach(function() {
+        rimraf.sync(ROOT);
+        fs.mkdirpSync(ROOT);
+
+        fixturify.writeSync(ROOT, {
+          'hello.txt': "Hello, World!\n",
+          'goodbye.txt': 'Goodbye, World\n',
+          'a': {
+            'foo': {
+              'one.js': '',
+              'one.css': '',
+            },
+            'bar': {
+              'two.js': '',
+              'two.css': '',
+            }
+          },
+          'b': {
+            'four.txt': '',
+          },
+        });
+        debugger;
+        tree = new FSTree({
+          entries: walkSync.entries(ROOT),
+          root: ROOT,
+          srcTree: true,
+        });
+      });
+
+      afterEach(function() {
+        fs.removeSync(ROOT);
+      });
+
+      it('include filters with parent dir', function() {
+        tree.include = ['**/one.css'];
+        let changes = tree.changes();
+
+        expect(changes).to.have.property('length', 3);
```

Comment:
Can we update these to do one `deeply.equal` and compare against the exact expected object, instead of all these individual assertions...